Framework/Algorithms/src/PDCalibration.cpp

```diff
@@ -261,6 +263,12 @@ void PDCalibration::init() {
                   "Range for allowable DIFA from calibration (default "
                   "is all)");
 
+  declareProperty("UseChiSq", false,
+                  "By default the square of the peak height is used as weights "
+                  "in the least-squares fit to find the diffractometer "
+                  "constants, if UseChiSq is true then the square of "
```

Comment:
It uses (1/centre_error^2) as the weight right? I guess its implied by using chi squared, but may be nice to clarify that here

Reply:
Agreed, I have clarified this now - thanks!